### DISCUSSION TITLE
Support for 30kHz scs on srsUE

### DISCUSSION THREAD

**Discussion by jboateng1** [srsran/srsRAN_Project#74] [General Help] [answered]
Folks,

Here at Iowa State, we are building a **large scale outdoor experimental testbed** (which is a $16M project in central Iowa) that features srsRAN 5G stack for our SDR platforms (N320 for gNB and B210 for UE) for the 5G research community. We have had good success with establishing a connection with the recent srsRAN gNB for band n3 and 15kHz FDD in our indoor sandbox- thanks to the srs team.

To be able to deploy this outdoor to support practical 5G experiments with our SDRs and TDD frontends(PA+LNA), we may be required to run srsUE at TDD frequencies like those in n78. 

- Does the srsRAN team has any timelines/roadmaps as to when to support this in srsUE?
- Also, even with the current srsRAN gNB and srsUE code, I wonder if folks may assist whether it is possible to modify the gNB code to support 15kHz for band 78 to connect with srsUE 15kHz numerology? If there is any possibility to modify both gNB and UE code to fully support TDD, we would be extremely happy to learn.

**Comment by ismagom** [comment, marked as answer]
Hi, 

Unfortunately there are no plans to support 30 KHz in srsUE. 

For your outdoor plans, do you need to use an SDR-based UE? If you don't need that part, srsGNB can support n78 bands. 

If you need to use srsUE, then the starting point would be to modify srsRAN gNB and srsUE to support 15 KHz in band 78. This is possible in theory, although not something we have tested.
